lib/rules/avoid-inline-spacing.json

```diff
@@ -0,0 +1,12 @@
+{
+	"id": "avoid-inline-spacing",
+	"selector": "[style]",
+	"tags": ["wcag21", "wcag1412"],
+	"metadata": {
+		"description": "Ensure text spacing is not affected by inline spacing styles that affects CSS specificity",
+		"help": "Remove inline styles with '!important' that affect text spacing, as overriding this is not supported by most browsers"
```

Comment:
Did a little more editing. I think the pass / fail ones are good. Can you change help/description to:


Description: Ensure that test spacing set through style attributes can be adjusted with custom stylesheets
Help: Inline text spacing must be adjustable with custom stylesheets

The browser thing is part of the fail message, we don't need to repeat that.